docs/root/intro/arch_overview/websocket.rst

```diff
@@ -48,7 +48,7 @@ In this case, if a client is for example using WebSocket, we want the Websocket
 upstream server functionally intact, which means it needs to traverse the HTTP/2 hop.
 
 This is accomplished via
-`extended CONNECT <https://tools.ietf.org/html/draft-mcmanus-httpbis-h2-websockets>`_ support. The
```

Comment:
Thanks for the update!

mind removing "alpha" above?  I think that was there due to the IETF work (and so nghttp2 support) not having landed.

Reply:
Done.